config/routes.rb

```diff
@@ -1,9 +1,22 @@
 Rails.application.routes.draw do
+  root 'home#index'
+
+  # routes for donating
   get 'donate/new' => 'donate#new', :as => 'new_donation'
   get 'donate/success' => 'donate#success', :as => 'successful_donation'
   get 'donate/error' => 'donate#fail', :as => 'failed_donation'
   get 'donate/:stripeToken/:amount' => 'donate#create', :as => 'create_donation'
+
+  # routes for applicants
+  get 'apply/new' => 'apply#new', :as => 'new_apply'
```

Comment:
Rename it new_application; it makes more sense.

Reply:
Rails generates an "application_controller" by default for every app, so that name was taken.

Reply:
Just to clarify I meant just to change the 'new_apply' string named that would become new_apply_path or as I'm suggesting new_application_path, I'm not sure if that was clear.  I don't think it will break it but I may be wrong.

Reply:
I'm against changing it because it breaks convention over configuration (the controller is called Apply). And if we do want to do something inside ApplicationController (which I think is like a global controller) we'd have to rename stuff there. We can always change it later (it's only lines in this file) if other teammates want to.

Reply:
I don't think it breaks convention over configuration.  You can see I did it for literally all the other paths (lines 5-8, 14) and it's not caused any problems.  As far as I know there is no place where it assumes the name will be new_apply.

Reply:
renamed to ```xx_application```